babeljs/jscontext.py

```diff
@@ -0,0 +1,102 @@
+try:
+    from py_mini_racer import py_mini_racer
+except ImportError:
+    py_mini_racer = None
+
+try:
+    from shutil import which
+except ImportError:
+    from distutils.spawn import find_executable as which
+
+import subprocess
+import json
+import tempfile
+import random
+import os
+
+
+if py_mini_racer:
+    ContextException = py_mini_racer.MiniRacerBaseException
+else:
+    class ContextException(Exception):
+        pass
+
+
+class NodeException(ContextException):
+    pass
+
+
+def get_context(use_mini_racer=None):
```

Comment:
`use_mini_racer=False` ?

Reply:
The idea was to have `None` mean "autodetect".  Perhaps it would be clearer to make this a three-state variable instead of a boolean?

e.g.
```python
transformer.transform_string('const a = () => 233') # use default (autodetect)
transformer.transform_string('const a = () => 233', runtime='auto') # autodetect
transformer.transform_string('const a = () => 233', runtime='node') # use Node
transformer.transform_string('const a = () => 233', runtime='miniracer') # use PyMiniRacer
```

Reply:
@sheppard Since this, named `runtime` is better. `use_mini_racer` is more like a boolean value